config/config.go

```diff
@@ -36,6 +36,11 @@ const (
 	EnvGrafanaVarServiceSource = "GRAFANA_VAR_SERVICE_SOURCE"
 	EnvGrafanaVarServiceDest   = "GRAFANA_VAR_SERVICE_DEST"
 
+	EnvJaegerDisplayLink      = "JAEGER_DISPLAY_LINK"
```

Comment:
Apparently we won't want to turn off jaeger link, as we want to fully leverage it, unlike Grafana. So I think this setting is not necessary.

Reply:
Ok I am gonna remove it